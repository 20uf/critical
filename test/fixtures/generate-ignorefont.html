<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title>Font-face</title>
    <link rel="stylesheet" href="styles/font.css"/>
</head>
<body>
    <h1>should be styled by @font-face</h1>

</body>
</html>
