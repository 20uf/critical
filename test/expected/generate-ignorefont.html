<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title>Font-face</title>
    <style type="text/css">
body {
  font-family: 'PT Sans', sans-serif;
}
</style>
<script id="loadcss">
(function(u,s){function loadCSS(e,n,t){"use strict";var o=window.document.createElement("link"),d=n||window.document.getElementsByTagName("script")[0],i=window.document.styleSheets;return o.rel="stylesheet",o.href=e,o.media="only x",d.parentNode.insertBefore(o,d),o.onloadcssdefined=function(e){for(var n,t=0;t<i.length;t++)i[t].href&&i[t].href===o.href&&(n=!0);n?e():setTimeout(function(){o.onloadcssdefined(e)})},o.onloadcssdefined(function(){o.media=t||"all"}),o}for(var i in u){loadCSS(u[i],s);}}(['styles/font.css'],document.getElementById("loadcss")));
</script>
<noscript>
<link rel="stylesheet" href="styles/font.css">
</noscript>
</head>
<body>
    <h1>should be styled by @font-face</h1>

</body>
</html>
